sdks/js/src/index.ts

```diff
@@ -1254,7 +1313,7 @@ export class DustAPI {
           url,
           duration,
           connectorsError: err,
-          error: sanitizedError(e),
+          error: e instanceof Error ? { name: e.name, message: e.message } : e,
```

Comment:
why not sanitizeError?

Reply:
sanitize error was doing nothing except remove the config for axios errors. So no it's just really not doing anything. Let me change this code to just put the error as is